<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavePvlDialog</class>
 <widget class="QDialog" name="SavePvlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Additional Information</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large;&quot;&gt;Bilateral filtering is a non-linear filtering technique. It extends the concept of Gaussian smoothing by weighting the filter coefficients with their corresponding relative pixel intensities. Voxels that are very different in intensity from the central pixel are weighted less even though they may be in close proximity to the central voxel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Voxel Unit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="voxelUnit">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>no units</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>angstrom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nanometer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>micron</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>millimeter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>centimeter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>meter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kilometer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>parsec</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kiloparsec</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>53</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5" rowspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Voxel Size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="voxelSize">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>1 1 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>53</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="description">
     <property name="text">
      <string>Information about volume</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Volume Filter</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="volumeFilter">
     <item>
      <property name="text">
       <string>no filter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3x3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5x5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7x7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9x9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11x11</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SavePvlDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SavePvlDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
